devnet-sdk/testing/testlib/validators/validators_test.go

```diff
@@ -292,21 +294,10 @@ func (m *mockNode) PendingNonceAt(ctx context.Context, addr common.Address) (uin
 	return 0, fmt.Errorf("not implemented")
 }
 
-func (m *mockNode) BlockByNumber(ctx context.Context, number *big.Int) (*ethtypes.Block, error) {
-	isIsthmusActivated, err := IsForkActivated(m.chainConfig, rollup.Isthmus, 100)
-	if err != nil {
-		return nil, err
-	}
-	var blockConfig *ethtypes.BlockConfig
-	if isIsthmusActivated {
-		blockConfig = ethtypes.DefaultBlockConfig
-	} else {
-		blockConfig = ethtypes.IsthmusBlockConfig
-	}
-
-	return ethtypes.NewBlock(&ethtypes.Header{
-		Time: 100,
-	}, &ethtypes.Body{Withdrawals: []*ethtypes.Withdrawal{}}, nil, nil, blockConfig), nil
+func (m *mockNode) BlockByNumber(ctx context.Context, number *big.Int) (eth.BlockInfo, error) {
+	header := ethtypes.Header{Time: 100}
+	info := eth.HeaderBlockInfo(&header)
+	return info, nil
```

Comment:
I believe failing to update the block header with Isthmus settings will break Isthmus tests.

Reply:
AFAIK the test does not use the config settings, and tests still passes.

Reply:
Hmm I'm getting errors running `go test ./devnet-sdk/...`. Maybe they're not included in CI yet or I have some local changes.

Reply:
Could you try running?

Reply:
@teddyknox so you are expecting the `devnet-sdk/testing/testlib/validators` to fail? I force-pushed to retouch other failing test. Can you try again?

Reply:
Yeah these test are not included in the CI. The previous failing tests are not related with this mockNode diff.

Reply:
Would be nice to have the CI run those tests to ensure this PR is ok before merging. What do you think?